### DISCUSSION TITLE
Nominate @robdimsdale as a Tooling Maintainer

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#166] [Tooling Team]
RE: https://github.com/orgs/paketo-buildpacks/teams/tooling-maintainers/discussions/2

I am nominating Rob Dimsdale-Zucker as a Tooling Maintainer. He has already expressed interest and demonstrated examples of his work in the linked discussion post.

**Comment by ryanmoran** [comment]
With that, we have a majority of 👍 votes from existing maintainers. Congrats @robdimsdale on your new maintainership!
